c/flake.nix

```diff
@@ -0,0 +1,44 @@
+{
+  description = "Aux template for C project";
+
+  inputs.nixpkgs.url = "github:auxolotl/nixpkgs/nixos-unstable";
```

Comment:
```suggestion
  inputs.nixpkgs.url = "github:auxolotl/nixpkgs";
```
we aren't actually updating branches besides master currently

Reply:
master template use it too.

Reply:
this should changed as well

Reply:
This might need it's own dedicated issue